cpu/atmega_common/periph/rtt.c

```diff
@@ -18,19 +18,33 @@
  *              oscillator.
  *
  * @author      Matthew Blue <matthew.blue.neuro@gmail.com>
+ * @author      Alexander Chudov <chudov@gmail.com>
  *
+ * For all atmega except rfr2
  * In order to safely sleep when using the RTT:
  * 1. Disable interrupts
  * 2. Write to one of the asynch registers (e.g. TCCR2A)
  * 3. Wait for ASSR register's busy flags to clear
  * 4. Re-enable interrupts
  * 5. Sleep before interrupt re-enable takes effect
  *
+ * For MCUs with MAC symbol counter (atmega*rfr2):
+ * The MAC symbol counter is automatically switch to TOSC1 32.768kHz clock
+ * when transceiver or CPU is going to sleep. The MAC symbol counter is
+ * sourced by 62.500 kHz derived from 32.768kHz TOSC1 or 16 MHz system clock
+ * For current implementation symbol counter is always use 32.768kHz TOSC1 clock
+ * For alarms the SCOCR2 register is used.
+ *
+ * SCCR0 is defined if an MCU has MAC symbol counter
+ *
  * @}
  */
 
 #include <avr/interrupt.h>
 
+#ifdef SCCR0
+#include "byteorder.h"
```

Comment:
Since `#ifdef SCCR0` is not immediately obvious throughout the code, maybe add a 
```C
#define RTT_BACKEND_SC (1)
#else 
#define RTT_BACKEND_TIM2 (1)
#endif 
```
here so you can use `#ifdef RTT_BACKEND_SC` / `#ifdef RTT_BACKEND_TIM2` throughout the code. (Not sure if the second define is really necessary or if it would add confusion)